docs/advtools.rst

```diff
@@ -35,3 +35,18 @@ enemies will gain the ``slay`` and ``kill`` keywords. It will also add
 additional conversation options for asking whereabouts of your relationships --
 in vanilla, you can only ask whereabouts of historical figures involved in
 rumors you personally witnessed or heard about.
+
+``advtools.fastcombat``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When enabled, this overlay will allow you to skip most combat animations, 
+including the whooshes and projectiles travelling through the screen. It will
+also let you skip the announcements window if "More" button is active,
+scrolling you to the very bottom with the first press, and skipping the window
+entirely with the second press. This drastically speeds up combat while still
+being intuitive enough - you can still be slow and methodical if you choose
+not to skip. Skip keys are left mouseclick, the SELECT button, the movement
+keys and combat-related keys that don't bring up its own menu (such as bump
+attack). If clicking to skip past combat, it will properly understand when
+you're clicking within the announcements box and not activate the script -
+only clicking OUTSIDE it will process the skipping.
```

Comment:
```suggestion
attack). If clicking to skip past combat, it will only skip the announcements
if you're clicking outside the announcements panel.
```